solutions/usaco-736.mdx

```diff
@@ -126,6 +126,72 @@ file_out.close()
 ## Solution 2 - Hashset
 
 <LanguageSection>
+					
+<CPPSection>
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(MN)$
+
+```cpp
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <vector>
+#include <algorithm>
+#include <map>
+
+using namespace std;
+
+int main() {
+
+    ifstream fin("cownomics.in");
+    ofstream fout("cownomics.out");
+
+    int N, M;
+    fin >> N >> M;
+    vector<string> spotty(N);
+    for (int i = 0; i < N; i++) fin >> spotty[i];
+    vector<string> plain(N);
+    for (int i = 0; i < N; i++) fin >> plain[i];
+
+    map<char, int> spottyCt;
+    map<char, int> plainCt;
+    int ans = 0;
+    
+    for (int i = 0; i < M; i++) {
+        
```

Comment:
there's really no need for all these blank lines

https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace